apps/prairielearn/src/lib/file-store.ts

```diff
@@ -120,28 +132,50 @@ export async function deleteFile(file_id, authn_user_id) {
 /**
  * Option of returning a stream instead of a file
  *
- * @param {number | string} file_id - The file to get.
- * @return {Promise<(import('stream'))>} - Requested file stream.
+ * @param file_id - The file to get.
+ * @return Requested file stream.
  */
-export async function getStream(file_id) {
+export async function getStream(file_id: number | string): Promise<ReadStream> {
   debug(`getStream(): file_id=${file_id}`);
   const file = await getFile(file_id, 'stream');
   return file.contents;
 }
 
+export async function getFile(
+  file_id: number | string,
+  data_type: 'stream',
+): Promise<{
+  contents: ReadStream;
+  file: any;
+}>;
+
+export async function getFile(
+  file_id: number | string,
+  data_type: 'buffer',
+): Promise<{
+  contents: Buffer;
+  file: any;
+}>;
 /**
  * Get a file from the file store.
  *
- * @param {number | string} file_id - The file to get.
- * @return {Promise<object>} An object with a buffer (of the file contents) and a file object.
+ * @param file_id - The file to get.
+ * @return An object with a buffer (of the file contents) and a file object.
  */
-export async function getFile(file_id, data_type = 'buffer') {
+export async function getFile(
+  file_id: number | string,
+  data_type = 'buffer',
+): Promise<{
+  contents: Buffer | GetObjectOutput['Body'] | ReadStream;
+  file: any;
```

Comment:
Bonus points if you update `queryZeroOrOneRowAsync` to `queryOptionalRow` with `FileSchema` so you can use the `File` type from `db-types.ts` instead of `any`.